Code/RDGeneral/RDLog.h

```diff
@@ -30,48 +30,67 @@ typedef boost::iostreams::stream<RDTee> RDTeeStream;
 class RDKIT_RDGENERAL_EXPORT rdLogger {
  public:
   std::ostream *dp_dest;
-  bool df_owner, df_enabled;
+  bool df_owner;
+  bool df_enabled;
 
+  std::ofstream *helperstream;
```

Comment:
please rename this to something `dp_teeHelperStream` so that it's consistent with our usual naming scheme

Reply:
Done.